adds/start.sh

```diff
@@ -54,11 +54,21 @@ configure_homeserver_yaml() {
 	mv ${ymltemp} "${filepath}"
 }
 
+configure_log_config() {
+        sed -i "s|.*filename:\s/homeserver.log|    filename: /data/homeserver.log|g" "/data/${SERVER_NAME}.log.config"
+}
+
```

Comment:
ok my bad then... i thought that the `generate` command should make a good first config creation. i did mine with that command and wondered why the logs are not in the right directory.